### DISCUSSION TITLE
Wikitext for <br>

### DISCUSSION THREAD

**Discussion by twMat** [TiddlyWiki/TiddlyWiki5#5332] [Ideas]
There is no native wikitext solution for a line break (right?) and we must type `<br>`. That html construct is just a tad to annoying to type and it is just ugly.

Can't it simply be something like:

`this is a line/`
`this is a new line`

...i.e a simple slash followed by a newline? Of course, the rendered result should hide the slash.

Or maybe even space+newline

`this is a line `
`this is a new line`

This looks great in edit mode exactly because you *don't* see anything and the actual line break (in edit mode) does make it distinctive, i.e it is not incidental.

**Comment by pmario** [comment]
have a closer look at: https://groups.google.com/g/tiddlywiki/c/rtTrJ4Iwj3s Space-Space-Newline plugin

**Reply by pmario** [reply]
space - space - newline is "compatible" with the CommonMark spec for markdown

**Reply by twMat** [reply]
Ha! I obviously missed it when casually playing with your plugins the other day. It seems to work very well and it's a small and independent plugin from what I can tell. Thank you!

BTW, there's probably a small misspelling towards end of the Demo GettingStarted tiddler. It states:

`\rules excpet ssnl`

I'm guessing it should read `except`
